core/src/main/java/com/linecorp/armeria/client/retry/RetryingHttpClientBuilder.java

```diff
@@ -73,19 +73,23 @@ public RetryingHttpClientBuilder useRetryAfter(boolean useRetryAfter) {
     }
 
     /**
-     * Sets the length of content to look up whether retry or not when you create this builder with a
-     * {@link RetryStrategyWithContent}. If the total length of content exceeds
-     * this and there's no retry condition matched, it will hand over the stream to the client.
-     * @param contentPreviewLength the content length to preview. Unlike other length related arguments,
-     *                             {@code 0} does not disable the length limit.
+     * Sets the length of content required to determine whether to retry or not. If the total length of content
+     * exceeds this length and there's no retry condition matched, it will hand over the stream to the client.
+     * Note that this property is useful only if you specified {@link RetryStrategyWithContent} when calling
+     * this builder's constructor. The default value of this property is {@link Integer#MAX_VALUE}.
```

Comment:
`{@literal #DEFAULT_CONTENT_PREVIEW_LENGTH}`?

Reply:
Change to `{@value ...}` Thank you~!